content/components/underline-nav.mdx

```diff
@@ -0,0 +1,82 @@
+componentId: underline_nav
+reactId: underline_nav
+figma: https://www.figma.com/file/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web?node-id=18843-67449&t=XqqSolC6AzRO9nae-11
+rails: https://primer.style/view-components/components/alpha/underlinenav/
+---
+
+# Underline nav
+
+The underline nav is used to provide navigation links to related views for the user's current context. It is typically used as primary navigation at the top of the page or as secondary navigation when nested in views with a [nav list](/components/nav-list).
+
+## Anatomy
+
+![Anatomy of the UnderlineNav component](https://user-images.githubusercontent.com/175638/186432408-c2479dd2-31fe-4261-9f9a-e67d49b28e72.png)
+
+1. **Item**: A navigation link.
+2. **Current item**: The current selection item is highlighted with an underline.
+3. **Label**: Describes the navigation item.
+4. **Leading icon (optional)**: An icon that identifies the navigation item.
+5. **Counter (optional)**: Displays a count next to the label.
+6. **Overflow menu**: A disclosure menu that displays additional items when there are too many to fit in the available space.
+
+## Content
+
+### Item
+
+The underline nav is composed of a list of items. Each item is a link that can be selected to navigate to a different page in the current context.
+
+#### Label text
+
+The label indicates the destination page of the link. Keep labels clear, short, and concise.
+
+#### Leading icons
+
+Use an Octicon to improve scannability of common items, but only when relevant. When adding leading icons, all items in the navigation should have one.
+
+Make sure that icons used are consistent across views, and that they are not repeated for different navigations across the product.
+
+#### Counters
+
+You may include a counter to indicate the number of related resources contained in the desitination page, such as the number of open issues.
+
+When loading multiple counters asynchronously, wait for all the data to be ready before displaying the counters, so you can avoid multiple layout shifts. Use the counter loading state while waiting for the data.
+
+## Layout
+
+### Placement
+
+The underline nav has two possible main placements depending on its intended usage:
+
+- As primary navigation: place it at the top of the page. Ensure any secondary and tertiary navigation is nested under the underline nav.
+- As secondary or tertiary navigation: place it above the main content of the.
+
+In either scenarios, avoid stacking multiple uderline navs to display multiple levels of navigation.
+
+### Alignment
+
+Align the the underline nav to the left and, ehen possible, make the component span the available width.
+
+### Responsive design
+
+Underline nav handles overflowing items automatically by collapsing them into a disclosure menu. When some of the items don't fit the available space, the component will first hide leading icons to economize space and then move items to the overflow menu one by one.
+
+## Accessibility
```

Comment:
I'm kind of surprised we don't treat this component semantically like tabs. I assume it's because of the overflow behavior?

Reply:
I talked to a11y about this, and I understand why we don't treat this semantically like tabs.